krun/util.py

```diff
@@ -180,3 +181,15 @@ def print_session_info(config):
             print("  %s" % k)
     else:
         print("  No keys skipped")
+
+
+def make_heat():
+    """A loop which spins, attempting to make heat.
+
+    Used when the CPU is too cool to run a benchmark."""
+
+    until = time.clock() + 1  # in roughly one second
+    while True:
+        # Waste cycles
+        if time.clock() > until:
```

Comment:
I can't guarantee how many times a second the syscall will be called, it depends upon how fast the system can execute python code.

So I think you are suggesting that we should add a loop like `for i in xrange(1000000): pass` or some such? Am I right?